README.md

```diff
@@ -2,16 +2,14 @@
 Author: inblockio / 23. August 2021
 
 Installation for Linux (Tested in Ubuntu 20.04) of your Personal Knowledge Container\
-This should work for other environments like Windows and Mac if the dependancies are fullfilled.
+This should work for other environments like Windows and Mac if the requirements are met.
```

Comment:
Shouldn't there be an extra empty line before the new subsection?

Reply:
I'd like to rewrite this entire README but it's not in scope for this PR. Let's plan later to make our README files across repos more consistent so we can tackle all README files at the same time.